ct/install_hpipm.sh

```diff
@@ -6,6 +6,10 @@ cd /tmp
 git clone https://github.com/giaf/blasfeo.git
 cd /tmp/blasfeo
 git checkout 50dc49e # we currently are on this release
+echo -e "\n\n\n======CONFIGURATION STEP NECESSARY======\nBefore continuing, open and modify /tmp/blasfeo/Makefile.rule and set the target to your architecture"
```

Comment:
@CedricPerottet actually I would find it quite useful to have this hint given in the documentation. Maybe we could just include a sentence pointing out that you can modiy that target architecture, if you run into a compilation problem with blasfeo?

Reply:
Ok, I will put that in the documentation then and do a new pull request. 

What I don't understand though is why the continuous integration failed. I saw that install_hpipm.sh was not called in the .github workflow, so I don't understand why the test didn't pass ?

Reply:
I restarted the jobs -- it should not have to do anything with your commit!

Reply:
One thought here: the proposed changes unfortunately enfore user interaction -- it might make sense to display the message only if the line `make static_library` fails. It might be worth checking if the `make static_library` [returns with a proper exit code](https://bencane.com/2014/09/02/understanding-exit-codes-and-how-to-use-them-in-bash-scripts/), which could be used to improve the user interaction here!

Reply:
Unfortunately, the library is successfully built and installed, it's only when calling the solver that there is a crash. I removed the modification in the bash, and added a warning note in the documentation instead, I will do the pull request now !